### PR TITLE
Fix: Timestamp in Onboarding data [ED-7140]

### PR DESCRIPTION
**Pull request by guyc-E**

### CODE REVIEW

**Review by matipojo — CHANGES_REQUESTED** [review]
Please cover with tests

File: assets/dev/js/utils/time.js [hunk 1]
```diff
@@ -16,12 +16,15 @@ export default function getUserTimestamp() {
...
-		sign = 0 <= timezoneOffsetInHours ? '+' : '',
-		addedZero = 10 < ( timezoneOffsetInHours ) ? '0' : '';
+		sign = 0 <= decimalTimezoneOffset ? '+' : '-',
+		hours = Math.abs( Math.floor( decimalTimezoneOffset ) ),
+		minutes = Math.abs( decimalTimezoneOffset % 1 ) * 60,
+		addZeroToHour = 10 > ( hours ) ? '0' : '',
+		addZeroToMinutes = 10 > ( minutes ) ? '0' : '';
 
-	const formattedTimezoneOffset = sign + addedZero + timezoneOffsetInHours + ':00';
+	const formattedTimezoneOffset = sign + addZeroToHour + hours + ':' + addZeroToMinutes + minutes + ':00';
```
Comment (udidol) [inline assets/dev/js/utils/time.js:27]:
```suggestion
	const formattedTimezoneOffset = sign + addZeroToHour + hours + ':' + addZeroToMinutes + minutes;
```

The `:00` represented minutes, so now that you are adding the minutes with variables, this suffix is redundant.

**Review by udidol — CHANGES_REQUESTED** [review]

**Review by udidol — DISMISSED** [review]

**Review by udidol — DISMISSED** [review]

File: tests/jest/unit/assets/dev/js/utils/time.test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Arrange.
+import getUserTimestamp from 'elementor-utils/time';
+
+describe( 'Time Utility', () => {
```
Comment (matipojo) [inline tests/jest/unit/assets/dev/js/utils/time.test.js:4]: Please fix the code and the test according to:
https://www.toptal.com/qa/how-to-write-testable-code-and-why-it-matters#:~:text=Testable%20and%20Untestable%20Code

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/jest/unit/assets/dev/js/utils/time.test.js (hunk 2 above)
Comment (udidol) [reply tests/jest/unit/assets/dev/js/utils/time.test.js:3]: Done

**Pull request merged**